src/app.ts

```diff
@@ -29,30 +29,44 @@ yargs(hideBin(process.argv))
                     type: "string",
                     description: "Output file path",
                 })
-                .option("parse-dependencies", {
-                    type: "boolean",
-                    default: false,
-                    description:
-                        "Flag to enable dependency parsing (dependencies will be appended to the output file)",
-                })
                 .option("exclusions", {
                     alias: "e",
                     type: "string",
                     description: "Exclude folders from scanning for files (comma separated list)",
                     default: "node_modules,.idea,dist,build,out,vendor",
                 })
-                .option("compress", {
-                    alias: "c",
+                .option("parse-h-as-c", {
+                    alias: "hc",
                     type: "boolean",
-                    description: "output .gz-zipped file",
+                    description: "Parse all .h files as C instead of C++ (defaults to C++)",
                     default: false,
                 })
+                .option("parse-some-h-as-c", {
```

Comment:
We should provide a usage example in the readme and explain if it the option applies recursively for the specified paths.

Reply:
It does, just like the "exclusions" option. Speaking of which, we should probably also include an usage example, etc., for that option?

Reply:
Yes, but I think the exclusion feature is not really nice right now. We should rework this to accept exclusion patterns. But this is out of scope here.

Reply:
Ok. Then for now, I will document that for both. We should remember to also change the parse-h-as-c option to also accept patterns once we change that for exclusions.

Reply:
Documentation for all command line options added to Readme